app/main.py

```diff
@@ -1,6 +1,20 @@
-# write your imports here
+from app.people.cinema_staff import Cleaner
+from app.cinema.bar import CinemaBar
+from app.cinema.hall import CinemaHall
+from app.people.customer import Customer
 
 
-def cinema_visit(customers: list, hall_number: int, cleaner: str, movie: str):
-    # write you code here
-    pass
+def cinema_visit(
+        customers: list[dict],
+        hall_number: int,
+        cleaner: str,
+        movie: str,
+) -> None:
+    customer_list = [Customer(**person) for person in customers]
```

Comment:
```suggestion
    customers_instances = [Customer(**person) for person in customers]
```
suffix _list is redundant